lua/starfall/sflib.lua

```diff
@@ -892,6 +892,22 @@ function SF.CheckMaterial(material)
 	return mat
 end
 
+
+function SF.CheckModel(model, player)
+	if #model > 260 then return false end
+	model = SF.NormalizePath(string.lower(model))
+	if util.IsValidModel(model) and util.IsValidProp(model) then
+		if player and player:IsValid() then
```

Comment:
❔ I'm curious as to why you don't like `IsValid(player)`?

Reply:
global IsValid uses a lot more CPU

Reply:
gotcha